xml/System/MemoryExtensions.xml

```diff
@@ -318,11 +331,11 @@
         <Parameter Name="startIndex" Type="System.Index" Index="1" FrameworkAlternate="netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="array">To be added.</param>
-        <param name="startIndex">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of the array.</typeparam>
+        <param name="array">The array to convert.</param>
+        <param name="startIndex">The first position of the array.</param>
+        <summary>Creates a new memory region over the portion of the target array starting at a specified index to the end of the array.</summary>
+        <returns>The memory representation of the array.</returns>
```

Comment:
@carlossanlop here is example of the place I was referring to.

Reply:
Got it. I'll make the modification.

```suggestion
        <returns>The memory representation of the whole or part of the array.</returns>
```

Reply:
@tarekgh there are some `<returns>` that instead of saying "representation of the array", they say "representation of the segment". Would you like me to alter those ones as well to "representation of the whole or part of the segment"?

Reply:
No, the instances saying ```"The span representation of the array segment``` is fine.

Reply:
Great, thank you. Then @rpetrusha this PR has finished baking.

Reply:
In that case, now that we finally have a successful build, I'll merge.